### PR TITLE
Add Onedata and CernVM-FS support

### PR DESCRIPTION
**Pull request by mtangaro**
Onedata and cvmfs support

New variables:
refdata_repository_name: This variable set the Onedata space name or cvmfs repository name or the download subdirectory.
refdata_provider_type: 'onedata' (default)
  1. onedata ------> Onedata space with reference data is mounted
  2. cvmfs --------> CernVM-FS repository with reference data is mounted
  3. download -----> Reference data download

- Onedata input options:
refdata_provider_type: 'onedata'
refdata_provider: Oneprovider url
refdata_token: Onedata access token

- CernVM-FS input options:
refdata_provider_type: 'cvmfs'
refdata_cvmfs_server_url: cvmfs url (stratum zero or replica).
refdata_cvmfs_key_file: ssh public kay to access the data
refdata_cvmfs_proxy_url: proxy url, if proxy enabled (default is DIRECT)
refdata_cvmfs_proxy_port: if a proxy url is specified we need the port.

- Download input options (deprecated):
refdata_provider_type: 'download' Enable ref. data download

### CODE REVIEW

File: examples/galaxy_tosca.yaml [hunk 1]
```diff
@@ -90,6 +122,15 @@ topology_template:
       properties:
         reference_data: { get_input: reference_data }
         flavor: { get_input: flavor }
+        refdata_repository_name: { get_input: refdata_repository_name }
+        refdata_provider_type: { get_input: refdata_provider_type }
+        refdata_provider: { get_input: refdata_provider }
+        refdata_token: { get_input: refdata_token }
+        refdata_cvmfs_server_url: { get_input: refdata_cvmfs_server_url }
+        refdata_cvmfs_repository_name: { get_input: refdata_cvmfs_repository_name }
```
Comment (micafer) [inline examples/galaxy_tosca.yaml]: Hi @mtangaro,
The property "refdata_cvmfs_repository_name" does not exist in the GalaxyReferenceData node type.

Comment (mtangaro) [inline examples/galaxy_tosca.yaml]: Sorry @micafer,
I've added it now.
Marco

Comment (micafer) [inline examples/galaxy_tosca.yaml]: There is another error, there in no input with name: refdata_cvmfs_repository_name.

Comment (mtangaro) [inline examples/galaxy_tosca.yaml]: Is it the same missing input as before?

Comment (micafer) [inline examples/galaxy_tosca.yaml]: Yes, but in this second case the problem is that refdata_cvmfs_repository_name does not appear as an input.

Comment (mtangaro) [inline examples/galaxy_tosca.yaml]: fixed and pushed (at least I hope).

**Pull request merged**